src/core_ocean/mode_init/Registry.xml

```diff
@@ -20,64 +20,64 @@
+        possible_values="'full_cell' or 'partial_cell'"
+  />
+  <nml_option name="config_min_pc_fraction" type="real" default_value="0.10" units="unitless"
+        description="Determines the minimum fraction of a cell altering the initial conditions can create."
+        possible_values="Any real between 0 and 1."
+  />
+</nml_record>
+<nml_record name="init_setup" mode="init">
+  <nml_option name="config_init_configuration" type="character" default_value="none" units="unitless"
+        description="Name of configuration to create."
+        possible_values="Any configuration name"
+  />
+  <nml_option name="config_expand_sphere" type="logical" default_value=".false." units="unitless"
+        description="Logical flag that controls if a spherical mesh is expanded to an earth sized sphere or not."
+        possible_values=".true. or .false."
+  />
+  <nml_option name="config_realistic_coriolis_parameter" type="logical" default_value=".false." units="unitless"
+        description="Logical flag that controls if a spherical mesh will get realistic coriolis parameters or not."
+        possible_values=".true. or .false."
+  />
+  <nml_option name="config_write_cull_cell_mask" type="logical" default_value=".true." units="unitless"
+        description="Logicial flag that controls if the cullCell field is written to output."
+        possible_values=".true. or .false."
+  />
+  <nml_option name="config_vertical_grid" type="character" default_value="uniform" units="unitless"
+        description="Name of vertical grid to use in configuration generation"
+        possible_values="'uniform', '60layerPHC', '42layerWOCE', '100layerE3SMv1', '1dCVTgenerator', ..."
+  />
+</nml_record>
+<nml_record name="CVTgenerator" mode="init">
+  <nml_option name="config_1dCVTgenerator_stretch1" type="real" default_value="1.0770" units="unitless"
+        description="Parameter for the 1D CVT vertical grid generator."
+        possible_values="Any positive non-zero integer."
+  />
+  <nml_option name="config_1dCVTgenerator_stretch2" type="real" default_value="1.0275" units="unitless"
+        description="Parameter for the 1D CVT vertical grid generator."
+        possible_values="Any positive non-zero integer."
+  />
+  <nml_option name="config_1dCVTgenerator_dzSeed" type="real" default_value="1.2" units="unitless"
+        description="Seed thickness of the first layer for the 1D CVT vertical grid generator."
+        possible_values="Any positive non-zero integer."
+  />
+</nml_record>
+<nml_record name="init_vertical_grid" mode="init">
+  <nml_option name="config_init_vertical_grid_type" type="character" default_value="z-star" units="unitless"
+        description="Which vertical grid to initialize with.  Without ice-shelf cavities (i.e. ssh=0 everywhere), 'z-star' and 'z-level' are the same."
+        possible_values="'z-star', 'z-level', or 'haney-number'"
+/>
```

Comment:
Oops, I don't think these whitespace changes should have happened.  Can you figure out which commit they're in, do and interactive rebase (`git rebase --interactive MPAS-Dev/MPAS-Model/ocean/develop`) and edit that commit to remove these changes (e.g. `git checkout src/core_ocean/mode_init/Registry.xml`)?

Reply:
Ok, I will do that. Thanks!